components/button/share/src/index.js

```diff
@@ -0,0 +1,83 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+import {Socialtwitter, Socialfacebook, Commentsquare, Envelopeclosed} from '@schibstedspain/sui-svgiconset'
+
+const getOnClickHandle = src => () => window.open(src)
+
+const parseHttpSchemeOfUrl = (urlToEval) => {
+  const isUrlValid = urlToEval.match(/http:\/\/|https:\/\//)
+  return !isUrlValid ? `http://${urlToEval}` : urlToEval
+}
+
+const getShareDefinition = (url, shareText, type) => {
+  const parsedShareUrl = url && parseHttpSchemeOfUrl(url)
+  switch (type) {
+    case 'facebook':
+      return {
+        src: `https://www.facebook.com/share.php?u=${parsedShareUrl}?opi=140&quote=${shareText}&tti=1&pagination=1&RowGrid=2`,
+        icon: Socialfacebook
+      }
+    case 'twitter':
+      return {
+        src: `http://twitter.com/share?url=${parsedShareUrl}&tti=1&pagination=1&RowGrid=2&text=${shareText}`,
+        icon: Socialtwitter
+      }
+    case 'whatsapp':
+      return {
+        src: `whatsapp://send?text=${shareText} - ${parsedShareUrl}`,
+        icon: Commentsquare
+      }
+    case 'email':
+      return {
+        icon: Envelopeclosed
+      }
+    default:
+      return {}
+  }
+}
+
+const ButtonShare = ({type, icon, onClick, shareText, url}) => {
+  const shareDefinition = getShareDefinition(url, shareText, type)
+  const Icon = icon || shareDefinition.icon
+  return <button
```

Comment:
Why not a link with href, target=_blank and let the native behaviour being executed ?
To me in makes no sense to use a button for a window.open.

Reply:
We didn't create this as a <a> because we thought in the email case. The email case in 'obra nueva' open a modal instead of redirect to an URL. This modal should be opened passing an onClick callback to the sharingButton.

It doesn't looks very pretty to opening a modal with an onClick handler in a <a> tag. What do you think, we keep the button, we change it to <a> or we make a mix between <a> and <button> tag wether if we have a onClick callback or not.

Reply:
It's ok. If you don't need the email implementation in your project you can skip it.

Reply:
We can do the mailto implementation is a good idea to have in that component.

Reply:
Done!